python/ml4ir/base/config/keys.py

```diff
@@ -135,7 +135,8 @@ class CalibrationKey(Key):
 
 class MonteCarloInferenceKey(Key):
     MONTE_CARLO_INFERENCE_TRIALS = "monte_carlo_inference_trials"
-    NUM_TRIALS = "num_trials"
+    NUM_TEST_TRIALS = "num_test_trials"
```

Comment:
@mohazahran Maybe `num_trials` and `num_trials_inference`?

And then if `num_trials_inference` is missing, you can use `num_trials`?

Reply:
@lastmansleeping I think being explicit of configuring the MC trails count for test and train is more clear and avoid confusion IMO

Reply:
[Nit] Okay, can you change to `num_trials_training` and `num_trials_inference`?